docs/modules/migrate/pages/dmt-dev-test.adoc

```diff
@@ -0,0 +1,115 @@
+= Try the DMT
+:description: You can use a Development cluster to try the DMT and see how a migration works before starting a Production cluster migration.
+
+{description}
+
+Before you start, make sure your development souce cluster is ready as follows:
+
+* Start the source cluster, as described in the xref:migrate:dmt-dev-test.adoc#start-the-source-cluster[] section below
+* Add data to the source cluster, as described in the xref:migrate:dmt-dev-test.adoc#add-data-to-source-cluster[] section below
+
+You can use the populated data to verify the migration, as described in the xref:migrate:dmt-dev-test.adoc#verify-the-migration[] section below.
+
+To complete the migration, follow the xref:migrate:dmt-migration.adoc#high-level-process[high-level process], omitting steps 3 and 5. 
+
+NOTE: You can skip steps 3, shutting down connected applications, and 5, updating your client configuration, as the test on a Development cluster only demonstrates how the data is migrated.
+
+== Start the Source Cluster
+
+You can start your source cluster in either of the following ways:
+
+* Using Docker, as described below. This is the recommended method
+* Downloading a version package and following the IMDG https://docs.hazelcast.com/imdg/4.2/getting-started[Quickstart]. For example, we used https://hazelcast.com/open-source-projects/downloads/archives/#hazelcast-imdg[Hazelcast IMDG version 4.2.7] for the samples provided in this section
```

Comment:
do we still use the word IMDG? Is it because the page is only available on 4.2? Is there a platform version of it? newer, for 5.x

Reply:
The example was to show the migration path - people can substitute their product/version and still follow the steps. I chose an older version because (a) it seems to be a common migration use case and (b) to highlight the steps required to update multiple versions.

As you say, it is fine, so I won't amend this.